mobile/src/constants/images/app.ts

```diff
@@ -49,6 +49,7 @@ import GoalsSVG from 'src/assets/images/app/goals-icon.svg';
 import GoalsActiveSVG from 'src/assets/images/app/goals-icon-active.svg';
 import GoalsIndicatorSVG from 'src/assets/images/app/goals-icon-indicator.svg';
 import GoalsIndicatorActiveSVG from 'src/assets/images/app/goals-icon-indicator-active.svg';
+import GoogleFitHealthPermPNG from 'src/assets/images/android_check.png'
```

Comment:
Naming here is off, the actual file google_fit_health_perm.png in unused while android_check.png is being imported as GoogleFitHealthPermPNG